Docs: Fix incorrect Docker autodiscover and add_docker_metadata default config value in documentation (#48011)
elastic/beats: docs/reference/filebeat/add-docker-metadata.md

:   (Optional) Specifies the mount point of the host’s filesystem, which can be used to monitor a host from within a container.

`labels.dedot`

> :   (Optional) Default to be true. If set to true, replace dots in labels with `_`.

Reviewer: ```suggestion
:   (Optional) Defaults to true. When true, replaces dots (`.`) in labels with underscores (`_`).
```
